tests/test_transaction.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+from happy_bank_core.logic.transaction import Transaction, TransactionException
+from happy_bank_core.logic.account import Account
+
+customer_1 = Account("id123", "John Doe", 100)
+customer_2 = Account("id321", "Johanna Doe", 300)
+
+
+class TestTransaction(unittest.TestCase):
```

Comment:
It's preferred to use pytest instead of unittest lib. You can use unittest.mock if you need but otherwise it's not necessary.